requirements.txt

```diff
@@ -2,8 +2,9 @@
 # pip install -r requirements.txt
 lxml
 requests
-rdflib
-rdflib-jsonld
+rdflib>=6.0.1; python_version>="3.7"
+rdflib<6.0.0; python_version<"3.7"
```

Comment:
There seems to be a limbo here for version 6.0.0.

Reply:
That's intentional: rdflib 5.0.0 is the latest available on python 3.6 and lower, and rdflib 6.0.1 is the latest available on python 3.7, there's no good reason you would need to stick with 6.0.0, which still requires the use of `rdflib-jsonld`.

Reply:
> there's no good reason you would need to stick with 6.0.0, which still requires the use of rdflib-jsonld.

I don't think 6.0.0 requires ``rdflib-jsonld``, see https://github.com/RDFLib/rdflib/blob/6.0.0/setup.py - so if 6.0.0 works fine for us, I'd rather not disallow it. Since extruct is a library, it makes sense to not be overly strict with what we require, to avoid conflicts with requirements of other packages.